code/datums/riding.dm

```diff
@@ -252,31 +252,195 @@
 					buckled_mob.pixel_x = 10
 					buckled_mob.pixel_y = 5
 
+//prototype repair vehicle
+/datum/riding/space/repair
+	keytype = null
+	vehicle_move_delay = 1
+	var/obj/machinery/repair_turret/turret = null
+	var/obj/vehicle/space/speedbike/repair/R = null
```

Comment:
# just use ridden this is snowflake dumb fucking code datums should be fucking modular you fuck cock monkey dumb fuck piece of fuck penis-chewing diarrhea squirt

fuck your cunt you shiteating cocksucker eat a dong you mass ending shitfuck eat penis in your fuckface and shit out abortions fuck your cunt and shit in your ass you cock fuck monkey fuck ass wanker from the depths of shit